packages/blade/src/components/Typography/Heading/Heading.stories.tsx

```diff
@@ -88,7 +89,7 @@ const AsPropTemplate: ComponentStory<typeof HeadingComponent> = (args) => {
         <Text as="span" weight="bold">
           Heading
         </Text>{' '}
-        component automatically changes the rendered HTML based on the{' '}
+        component automatically renders the respective `h*` tag based on the{' '}
```

Comment:
```suggestion
        component automatically renders the respective <Code>h*</Code> tag based on the{' '}
```

sorry I forgot we are not doing markdown 🤦

Reply:
Done. added for Title also.